lib/rules/web/admin_console/4.1/deployment.xyaml

```diff
@@ -27,4 +32,19 @@ search_image:
   action: clear_input_value
   action: set_input_value
   action: click_button
-
+search_nonexisted_image_and_check_message:
+  params:
+    search_content: non-existed
+    content: Could not load image metadata
+  action: search_image
+  action: check_page_contains
+search_and_deploy_image:
+  action: search_image
+  action: wait_form_loaded
+  action: click_deploy_button
```

Comment:
click_deploy_button could be replaced by click_button, pass button_text parameter.

Reply:
Cool, updated.

Reply:
PASS on Jenkins
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/44308/console